### DISCUSSION TITLE
v3.4.1

### DISCUSSION THREAD

**Discussion by andrew-platt** [OpenFAST/openfast#1448] [General]
# Release Notes

OpenFAST 3.4.1 is a minor release to revert a channel name change in AeroDyn 15.  This also contains minor fixes for documentation builds.

### Documentation
#1442 `[BugFix] Doxygen builds failing on rtd, and locally.   Documentation builds on readthedocs and local were failing due to a change in the backend of sphinx.  This fixes local builds, but does not fix readthedocs builds.  As a temporary workaround, doxygen is disabled on readthedocs.

### AeroDyn15
#1428 AD15: revert to Aero names for output channels (Fld is now an alias).  This fixes an issue introduced in v3.4.0 (#957)

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenFAST/openfast/releases/tag/v3.4.1'>v3.4.1</a>.</em>

**Comment by abhineet-gupta** [comment]
There was a small bug in OpenFAST v3.4.1 which caused abrupt termination of FAST.Farm simulation if the simulation runs for a long period of time.
This will be addressed in OpenFAST v3.5.1 in https://github.com/OpenFAST/openfast/pull/1684 .
However, I have created a patch for to fix this OpenFAST v3.4.1 in this [patch file](https://github.com/OpenFAST/openfast/files/12381539/patch.txt).

As this patch changes a registry file, please remember to recompile OpenFAST with GENERATE_TYPES option when compiling using cmake.
